packages/esm-appointments-app/src/appointments/common-components/appointments-table.component.tsx

```diff
@@ -126,14 +133,15 @@ const AppointmentsTable: React.FC<AppointmentsTableProps> = ({
                   size="lg"
                   kind="tertiary"
                   renderIcon={Download}
-                  onClick={() =>
-                    downloadAppointmentsAsExcel(
-                      appointments,
-                      `${tableHeading} Appointments ${formatDate(new Date(appointments[0]?.startDateTime), {
-                        year: true,
-                      })}`,
-                    )
-                  }>
+                  onClick={() => {
+                    const date = appointments[0]?.startDateTime
+                      ? formatDate(parseDate(appointments[0]?.startDateTime), {
+                          time: false,
+                          noToday: true,
+                        })
+                      : null;
```

Comment:
Tangential and maybe overkill, but this download is meant to download a set of appointments from a current day, so it doesn't really make sense to include the time in the filename, and if it's today, we don't want it to say "Today", we want it to display the actual date.  Also broke out getting the date from the method call because it was getting difficult to read.